<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowContent</class>
 <widget class="QWidget" name="ShowContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mostrar Conteúdo</string>
  </property>
  <widget class="QLabel" name="lbl_begin_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_begin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>14</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Comentário:</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>721</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_icon">
    <attribute name="title">
     <string>Notificação</string>
    </attribute>
    <widget class="QPlainTextEdit" name="txt_main_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_choose_video">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolher Vídeo</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="player_holder" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>721</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_content">
    <attribute name="title">
     <string>Mídias</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_behavior">
    <attribute name="title">
     <string>Comportamento</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>600</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/no.png</normaloff>:/i/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/check.png</normaloff>:/i/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
